src/vsr/replica.zig

```diff
@@ -1312,6 +1313,10 @@ pub fn ReplicaType(
             }
         }
 
+        pub fn invariants(self: *const Replica) void {
+            assert(self.journal.header_with_op(self.op) != null);
```

Comment:
Other ideas to verify:
- `self.log_view <= self.view`
- `log_view`/`view` are at least the superblock `vsr_state`'s views.
- maybe `op >= op_checkpoint` and `op <= op_prepare_max`?

Reply:
Yeah, there's a whole bunch of stuff I want to check here, but, the more conditions I add, the harder is to debug vopr failures, so I decided to start small!

Reply:
(what I _actually_ want to check specifically is that what Joran was talking about a while ago: that, if we have a prepare locally, we must be committing!)